src/components/Note/Note.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from 'react';
+import { Note, TextLink } from '@contentful/forma-36-react-components';
+
+export interface INoteProps {
+  type: 'primary' | 'positive' | 'negative' | 'warning';
+  error: Error;
+  resetErrorBoundary: () => void;
+}
+
+export function IxNote({
+  error,
+  type,
+  resetErrorBoundary,
+}: INoteProps): ReactElement {
+  const [message, _link, ...rest] = error.message.split('$');
+  const [link, title] = _link.split('|');
+  const linkTitle = title?.length ? title : link;
+  console.log(message, link, ...rest);
+
+  return (
+    <div className="ix-note" style={{ marginTop: 25 }}>
```

Comment:
weak opinion: we've been pretty good so far not styling anything inline so we should pull this out to stay consistent